doc/source/reference/configuration.rst

```diff
@@ -140,6 +142,43 @@ used internally.
 * ``supp_data_dir`` (default: ``/does/not/exist``): The default path certain
   submodules of yt look in for supplemental data files.
 
+
+.. _per-field-config:
+
+Available per-field Configuration Options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to customize the default behaviour of plots using per-field configuration.
+The default options for plotting a given field can be specified in the configuration file
+in ``[field_type.field_name]`` blocks. The available keys are
+
+* ``cmap`` (default: ``yt.default_colormap``, see :ref:`global-config`): the colormap to
+  use for the field.
+* ``log`` (default: ``True``): use a log scale (or symlog if linthresh is also set).
+* ``linthresh`` (default: None): the linear threshold for symlog plots.
```

Comment:
To be extra explicit we should say that if this is set the plot _will_ be logged.

Reply:
Oh wait, it will be symlogged if both?

Reply:
I have updated it. Tell me if that is clearer :)